corehq/apps/commtrack/templates/commtrack/manage/default_consumption.html

```diff
@@ -20,9 +20,11 @@
             <legend>{% trans 'Edit Default Monthly Consumption' %}</legend>
             {% crispy form %}
             <div class="form-actions">
-                <button type="submit" class="btn btn-primary">
-                    {% trans 'Update Default Consumption Info' %}
-                </button>
+                <div class="col-sm-12">
+                    <button type="submit" class="btn btn-primary">
+                        {% trans 'Update Default Consumption Info' %}
```

Comment:
please revisit the `FormActions` bit in the Django Forms Checklist https://www.commcarehq.org/styleguide/forms/#b3-migration